### ISSUE TITLE
CONTRIBUTING?

### ISSUE THREAD

**Issue by BarkleyBG** [jennybc/happy-git-with-r#50]
Could you add a CONTRIBUTING.md document with guidelines on how to contribute to this project? It's an exciting endeavor!

**Comment by jennybc** [comment]
Good idea. Do you mean mechanics or philosophy or both? Also, have you seen one for a similar project that you find helpful?

**Comment by BarkleyBG** [comment]
Both!

#### Philosophy
I'd like to know where you're looking for contributions, or where you're NOT looking for help. (Even if the guidelines simply say that anything is open to potential improvements)

#### Mechanics
- I saw a good one (and I can't remember where) that said "file the issue before working on the PR - it may be that the issue has already been worked out or otherwise not open for improvements at this time". I can keep an eye out for that, but I can't remember where it was.
- Microsoft's [nodejstools](https://github.com/Microsoft/nodejstools/blob/bab879e0913fab73da30020400b1f9dea22dacc9/.github/CONTRIBUTING.md) has a good start for their contributing mechanisms.
- David Robinson and Julia Silge's [Tidy Text Mining](https://github.com/BarkleyBG/tidy-text-mining/blob/master/CONDUCT.md) uses a nice pro-inclusiveness [template](http://contributor-covenant.org/version/1/0/0/)
